controllers/pulsarconnection_controller.go

```diff
@@ -81,8 +81,6 @@ type PulsarConnectionReconciler struct {
 // For more details, check Reconcile and its Result here:
 // - https://pkg.go.dev/sigs.k8s.io/controller-runtime@v0.11.0/pkg/reconcile
 func (r *PulsarConnectionReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
-	log := r.Log.WithValues("pulsarconnection", req.NamespacedName)
```

Comment:
Remove this line will also make the main PulsarConnection reconciler lose namespace/name in the log. 

I think you need add WithValues for the PulsarConnection reconciler too. i.e, around [here](https://github.com/streamnative/pulsar-resources-operator/pull/191/files#diff-c52731da76ee690e55193faa143fabf921b112117dc958937634de3d4ead8f1cR91)